### DISCUSSION TITLE
Add CSS variable for table header text color

### DISCUSSION THREAD

**Discussion by Jose-Verdu-Diaz** [pradyunsg/furo#801] [Ideas] [answered]
Furo currently offers a CSS variable for modifying the background colour of table headers:

https://github.com/pradyunsg/furo/blob/0439e84b3c2c5b168f22ea2e93c867fd37911dfa/src/furo/assets/styles/variables/_colors.scss#L76-L78

Currently, the table header text colour is set to `--color-foreground-primary`, however, this may lead to problematic colour combinations when modifying the header background colour.

It would come handy to add a new variable `--color-table-header-text` as follows:
```scss
--color-table-header-text: var(--color-foreground-primary);
```

In this way, the table header text could be overwritten from the sphinx config file.

**Comment by pradyunsg** [comment, marked as answer]
#818 filed for this.
